### DISCUSSION TITLE
Font

### DISCUSSION THREAD

**Discussion by InsightfulFuture** [beercss/beercss#192] [Ideas]
By inspect [the material 3 website](https://m3.material.io) I found that the default font family is actually `"Google Sans Text", sans-serif;` which looks different than the current system stack beercss is using, as that is inspired by roboto.

**Comment by acroyear** [comment]
"Google fonts are free to use for all purposes, including commercial use. However, there are some exceptions. For example, the Google Sans font can only be used in Google products by Google. This is because Google Sans is a custom font that was designed specifically for Google's products."

So this is not an issue. Google probably should fix that for their demo pages to use the font they recommend in their design standard rather than a font nobody else can use. From a design/development standpoint, that's a pretty bad overlook.

**Comment by acroyear** [comment]
https://m3.material.io/styles/typography/fonts are the fonts MD3 recommends (at least for Sans).

**Comment by laurentpayot** [comment]
From the [Material Design (2) blog](https://m2.material.io/design/typography/understanding-typography.html#system-fonts):
> Native system typefaces should be used for large blocks of text and any text below 14sp.
> Roboto is the default system font for Android. For platforms outside of Android and web products, use a system typeface that is preferred on that platform. For example, iOS applications should use Apple’s San Francisco font.

**Reply by acroyear** [reply]
That was material 2.

Material 3 (which beercss is aiming to implement) doesn't seem to make a statement like that per my read of the link above.

For myself, i prefer Roboto in all things because I'd rather have a consistent appearance across the platforms...but then i'm not writing large blocks of text outside of my promotional website.
